rpcs3/Emu/SysCalls/Modules/cellL10n.h

```diff
@@ -2,6 +2,8 @@
 
 namespace vm { using namespace ps3; }
 
+std::map<s32, std::pair<s32, s32>> converters;
+
```

Comment:
It's not only wrong because you forgot to remove it, it's bad idea to define variables in headers this way.